site/src/css/custom.css

```
@@ -184,20 +197,24 @@ div[class^='announcementBarContent'] a:hover {
 }
 
 .dropdown:not(.dropdown--nocaret) > .navbar__link::after {    /* dropdown menu caret */
+  background-image: var(--dropdown-arrow);
   border-color: currentColor transparent;
-  border-style: solid;
-  border-width: 0.4em 0.4em 0;
+  border-style: hidden;
   content: '';
-  margin-left: 0.5em;
+  width: 1em;
+  height: 1em;
   position: absolute;
-  top: 50%;
-  transform: translateY(-50%);
+  top: 48%;
 }
 
 .navbar .navbar__items {    /* shows full logo name on mobile */
   flex: auto;
 }
 
+.navbar__link {
+  font-family: 'Overpass-Bold';
```

```suggestion
  font-family: 'Overpass Bold';
```
Same change as above.